Source/Core/Core/HW/DSPHLE/UCodes/AXStructs.h

```diff
@@ -253,8 +253,9 @@ struct AXPB
   PBSampleRateConverter src;
   PBADPCMLoopInfo adpcm_loop_info;
   PBLowPassFilter lpf;
```

Comment:
Maybe we should document that `lpf` is a member thats treated differently in certain cases? Least we should do is put a comment similar to "skipped when writing to/reading from MRAM/ARAM for _certain_ AX UCodes"